use_sso_authentication/index.rst

```diff
@@ -0,0 +1,106 @@
+Use SSO authentication locally
+==============================
+
+This chapter describes how to enable the SSO authentication for development purpose.
+
+How SSO works ?
+---------------
+
+| SSO means `Single Sign-On`, their is different implementation/protocols in order to implement a SSO authentication mecanism.
+| We implemented the ``SAMLv2`` protocol as it becomes a standard in the industry.
+|
+
+    Security Assertion Markup Language 2.0 (SAML 2.0) is a version of the SAML standard for exchanging authentication and authorization data between security domains.
+    SAML 2.0 is an XML-based protocol that uses security tokens containing assertions to pass information about a principal (usually an end user) between a SAML authority, named an ``Identity Provider``, and a SAML consumer, named a ``Service Provider``.
+
+    -- Source `Wikipedia <https://en.wikipedia.org/wiki/SAML_2.0>`_
+
+The ``Identity Provider`` offer user authentication as a service, and the ``Service Provider`` could be considered as a client that will ask if a user is authenticated.
```

Comment:
"will ask to authenticate a user" seems more accurate to me.

Reply:
```suggestion
The ``Identity Provider`` offer user authentication as a service, and the ``Service Provider`` could be considered as a client that will ask to authenticate a user.
```